apps/heft/src/plugins/TypeScriptPlugin/TypeScriptBuilder.ts

```diff
@@ -114,8 +128,8 @@ interface IExtendedEmitResult extends TTypescript.EmitResult {
 const OLDEST_SUPPORTED_TS_MAJOR_VERSION: number = 2;
 const OLDEST_SUPPORTED_TS_MINOR_VERSION: number = 9;
 
-const NEWEST_SUPPORTED_TS_MAJOR_VERSION: number = 4;
-const NEWEST_SUPPORTED_TS_MINOR_VERSION: number = 8;
+const NEWEST_SUPPORTED_TS_MAJOR_VERSION: number = 5;
+const NEWEST_SUPPORTED_TS_MINOR_VERSION: number = 1;
```

Comment:
You might want to do 0 here, as 5.0's the one that's going to be released relatively soon (we just bumped main to 5.1 last night hence why nightlies are showing this). But, up to you.